sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -1125,7 +1125,7 @@ export class BlobClient extends StorageClient {
         abortSignal: options.abortSignal,
         leaseAccessConditions: options.conditions,
         modifiedAccessConditions: options.conditions,
-        onDownloadProgress: isNode ? undefined : options.onProgress,
```

Comment:
@XiaoningLiu do you know why download progress is not enabled for Node? I enabled it and my new tests in this PR passed.

Reply:
This is old code from V10

Reply:
I got it now.  For node case, we don't want to use the httpclient's progress event. Instead we will report the progress in the retriable stream.  Will revert.